.pre-commit-config.yaml

```diff
@@ -3,9 +3,6 @@ repos:
     rev: v4.3.0  # Use the ref you want to point at
     hooks:
       - id: trailing-whitespace
-      - id: detect-aws-credentials
```

Comment:
Seems to be broken, I'm removing it because there's a ticket in the back log to rethink these precommit hooks